### DISCUSSION TITLE
GPUDirect Stroage and pg version

### DISCUSSION THREAD

**Discussion by naturliche** [heterodb/pg-strom#760] [General]
Hello, I would like to ask two questions:
1: Does GDS only support NVIDIA's RTX and Quadro series? Does it not support GeForce RTX 3060?
2: Does pg-strom v3.5 support pg15? Because I saw that there is a PG_VERSIONS file in pg-strom v3.5, and 12-15 is written in it.

**Comment by ytooyama** [comment]
1. GDS doesn't work on GeForce GPU. This is a restriction by NVIDIA.
2. PG-Strom 3.5 also works with PostgreSQL 15. But, if you are a new user, we recommend PG-Strom 5.1.

**Reply by naturliche** [reply]
Thanks for your answer!
